test/ci/test-fsim-wget.sh

```diff
@@ -42,13 +42,15 @@ start_service_owner() {
   run_go_fdo_server owner ${owner_service} owner ${owner_pid_file} ${owner_log} \
     --owner-key="${owner_key}" \
     --device-ca-cert="${device_ca_crt}" \
-    --command-wget "${wget_test_url}"
+    --command-wget "${wget_source_url}"
 }
 
 run_test () {
   # Add the wget_httpd service defined above
   services+=("${wget_httpd_service_name}")
 
+  trap cleanup EXIT
```

Comment:
What's the reason of moving this line here?.

Currently it works this way:
- First we set `stop_servers` to be run before exiting. This way if the test fails we still have the logs available in `/tmp/go-fdo` and also allow us to run the test again.
- Once the tests have passed there's no need to keep `/tmp/go-fdo` so we set `cleanup` to be run before exiting.

Reply:
Ah - ok that makes sense. I moved it because the early return statement I added short-circuited the cleanup prior to registering the TRAP but as you point out we really want to keep those logs around in that case.  I'll revert this.